CityOfDetroit/bloom
# Support filtering via url params

## Issue
#240 

Addresses # (issue)


## Description

In the listings page, updates the url based on currently set filters. Also allows setting filters based on the url params.

NOTE: This change does NOT support multiple values per filter. That will come in a follow up.

## Type of change


## How Can This Be Tested/Reviewed?

Updating filters via the UI:
* In the public /listings page, use the filter modal to update a filter, and see the filters applied and the url change.

Updating filters via url:
* From the main /listing page (without any filters applied), add a filter to the url, e.g. `{listings_url}?page=1&neighborhood=Foster+City`

Also note that navigating via "back" or "forward" works as well.


## Checklist:
